gradle/gradle-publish.gradle

```diff
@@ -0,0 +1,77 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+
+ext["signing.keyId"] = System.getenv('SIGNING_KEY_ID') ?: ''
+ext["signing.password"] = System.getenv('SIGNING_KEY_PASSWORD') ?: ''
+ext["signing.secretKeyRingFile"] = System.getenv('SIGNING_KEY_FILE') ?: ''
+ext["SONATYPE_NEXUS_USERNAME"] = System.getenv('SONATYPE_NEXUS_USERNAME') ?: ''
+ext["SONATYPE_NEXUS_PASSWORD"] = System.getenv('SONATYPE_NEXUS_PASSWORD') ?: ''
+
+
+afterEvaluate {
+    android.libraryVariants.each { variant ->
+        publishing.publications.create variant.name, MavenPublication, {
+            groupId group
+            version project.version
+            artifactId project.name.toLowerCase()
```

Comment:
We might want some additional logic here for the artifact naming. Do we want dashes rather than all lower case for modules with two words (i.e. `core-payments` rather than `corepayments`)?

Reply:
I vote for using dashes in the artifactIDs.

Reply:
I added `toLowerCase` to transform `Card` and `Code` into `card`, `code`. If a module has two words, i would add the dash on the module name.

Reply:
I'd opt to pull the artifact names into their own gradle files along with the POM name and descriptions (since those should be module specific). Dashes in module names doesn't feel like Kotlin/Java convention to me. Thoughts?

Reply:
@sarahkoop agreed